<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuItem</class>
 <widget class="QWidget" name="MenuItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MenuItem
{
	border: 2px solid rgb(45,100,120);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="expandButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#expandButton
{
	background-color: rgb(45, 100, 120);
	font: 75 12pt &quot;微软雅黑&quot;;
	color: white;
	text-align: left;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="subItemWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color: rgb(85, 140, 160);
	font: 10pt &quot;微软雅黑&quot;;
	color: white;
	text-align: left;
	border:none;
	border-left: 1px solid rgb(65, 120, 140);
	border-right: 1px solid rgb(65, 120, 140);
	border-bottom: 1px solid rgb(65, 120, 140);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="managesystem.qrc"/>
 </resources>
 <connections/>
</ui>
